### PR TITLE
Increase the test coverage for tenable sc source files

### PR DESCRIPTION
**Pull request by tushar-balwani**
# Description

Added additional test cases to increase code coverage for TenableSC package

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:
* Python Version(s) Tested: 3.8.6
* Tenable.sc version (if necessary):

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by levwais — COMMENTED** [review]
please pull the two changes as suggested to another PR and have it approved and merged before we merge this one.

**Comment by adabala-durga** [comment]
> please pull the two changes as suggested to another PR and have it approved and merged before we merge this one.

I have raised PR #331 to address this, will remove file from this PR once #331  is merged.

File: tenable/sc/organizations.py [hunk 1]
```diff
@@ -539,11 +543,11 @@ def manager_edit(self, org_id, user_id, **kw):
             >>> secmngr = sc.organizations.manager_edit(1, 1,
             ...     username='updated')
         '''
-        payload = self._api.users._constructor(**kw)
-        return self._api.get('organization/{}/securityManager/{}'.format(
+        payload = self._api.users._constructor(**kwargs)
+        return self._api.patch('organization/{}/securityManager/{}'.format(
```
[the remark this answers is not in the document]
Comment (adabala-durga) [reply tenable/sc/organizations.py]: Done !! Moved this file to new PR #331

File: tenable/sc/organizations.py [hunk 2]
```diff
@@ -564,7 +568,7 @@ def manager_delete(self, org_id, user_id, migrate_to=None):
         if migrate_to:
             payload['migrateUserID'] = self._check('migrate_to', migrate_to, int)
 
-        self._api.get('organization/{}/securityManager/{}'.format(
+        self._api.delete('organization/{}/securityManager/{}'.format(
```
[the remark this answers is not in the document]
Comment (adabala-durga) [reply tenable/sc/organizations.py]: Done, thanks

**Review by levwais — APPROVED** [review]
looks good

**Pull request merged**